packages/apollo-engine-reporting/src/agent.ts

```diff
@@ -407,6 +408,13 @@ export class EngineReportingAgent<TContext = any> {
       );
     }
 
+
+    if (options.experimental_schemaReporting !== undefined) {
+      this.schemaReport = options.experimental_schemaReporting;
```

Comment:
should we also add an `options.schemaReporting` as well and deprecate the old one, or do you want to do that in another PR?

Reply:
I'd rather deprecate the field in a different PR. I think it should be in `2.15`